components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1237,14 +1293,15 @@ void ConversationDriver::RateMessage(
   // and probably a message uuid.
   uint32_t current_turn_id = turn_id + 1;
 
-  if (current_turn_id <= history.size()) {
+  if (current_turn_id <= history.size() && model.option->is_leo_model()) {
```

Comment:
This function already early returns if the model isn't Leo

Reply:
whoops, ok, removed the redundant check